frontend/public/components/sidebars/resource-sidebar-samples.tsx

```diff
@@ -68,6 +68,13 @@ const defaultSamples = ImmutableMap<GroupVersionKind, Sample[]>()
         id: 'docker-build',
         targetResource: getTargetResource(BuildConfigModel),
       },
+      {
+        title: 'Build from Devfile',
+        description:
+          'A Devfile build performs an image build using a devfile in the source repository or specified in build configuration.',
+        id: 'devfile-build',
+        targetResource: getTargetResource(BuildConfigModel),
+      },
```

Comment:
do we need to translate this as well?

Reply:
I intentionally didn't translate this, felt odd translating this one guy in this whole file. But probably should be 😞

Reply:
This file is now moved to shared -- I'm going to leave it and I'll swing in after this PR and do it